### PR TITLE
 Add link to open issues/feature requests in github

### PR DESCRIPTION
**Pull request by Flanker32**
Add links to report issues/request feature in github

Samples:
[report issue](https://github.com/Microsoft/azure-tools-for-java/issues/new?title=%5BIntelliJ%5D%5BReportedByUser%5D+Customer+Issues&labels=IntelliJ&body=%23%23%23%23+Setup%0A%0AIntelliJ+build+version%3A+2019.2+IC-192.5728.98%0AOS%3A+Windows+10%0AJDK%3A+Oracle+Corporation+1.8.0_211%0APlugin+version%3A+3.24.0%0A%0A%23%23%23%23++Describe+the+bug%0A%0AA+clear+and+concise+description+of+what+the+bug+is%2C+including+expected+behavior+and+actual+behavior%0A%0A%23%23%23%23++To+Reproduce%0A%0ASteps+to+reproduce+the+bug%0A%0A%23%23%23%23+Exception+or+Stack+Trace%0A%0AAdd+the+exception+log+and+stack+trace+if+available.%0A%0A%23%23%23%23+Screenshots%0A%0AAdd+screenshots+to+help+explain+your+problem+if+available.)
[request feature](https://github.com/Microsoft/azure-tools-for-java/issues/new?title=%5BIntelliJ%5D%5BReportedByUser%5D%5Bfeature-request%5D+Feature+Request&labels=IntelliJ%2Cfeature-request&body=%23%23%23%23++Is+your+feature+request+related+to+a+problem%3F+Please+describe%0A%0AA+clear+and+concise+description+of+what+the+problem+is.+Ex.+I%27m+always+frustrated+when+%5B...%5D%0A%0A%23%23%23%23++Describe+the+solution+you%27d+like%0A%0AA+clear+and+concise+description+of+what+you+want+to+happen.%0A%0A%23%23%23%23+Additional+context%0A%0AAdd+any+other+context+or+screenshots+about+the+feature+request+here.)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubFeatureRequestAction.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+package com.microsoft.azuretools.ijidea.actions;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.microsoft.azuretools.telemetry.TelemetryConstants;
+import com.microsoft.intellij.feedback.GithubIssue;
+import com.microsoft.intellij.feedback.NewGithubIssueAction;
+import com.microsoft.intellij.feedback.ReportableFeatureRequest;
+
+public class GithubFeatureRequestAction extends NewGithubIssueAction {
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubFeatureRequestAction.java]: Just make the classname more sense

```suggestion
public class NewFeatureRequestFeedbackAction extends NewGithubIssueAction {
```

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubIssueAction.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+package com.microsoft.azuretools.ijidea.actions;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.microsoft.azuretools.telemetry.TelemetryConstants;
+import com.microsoft.intellij.feedback.GithubIssue;
+import com.microsoft.intellij.feedback.NewGithubIssueAction;
+import com.microsoft.intellij.feedback.ReportableIssue;
+
+public class GithubIssueAction extends NewGithubIssueAction {
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubIssueAction.java]: Class named `NewCustomerBugFeedbackAction` or `NewCustomerIssueFeedbackAction` might be more sense

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/feedback/ReportableIssue.kt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.microsoft.intellij.feedback
+
+class ReportableIssue(shortMessage: String) : Reportable(shortMessage) {
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/feedback/ReportableIssue.kt]:
```suggestion
class ReportableBug(shortMessage: String) : Reportable(shortMessage) {
```

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubFeatureRequestAction.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubFeatureRequestAction.java]: Changed to `NewFeatureRequestFeedbackAction`

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubIssueAction.java (hunk 2 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubIssueAction.java]: Changed to `NewCustomerIssueFeedbackAction`

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/feedback/ReportableIssue.kt (hunk 3 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/feedback/ReportableIssue.kt]: I prefer to keep `ReportableIssue` to be consistent with `NewCustomerIssueFeedbackAction`

**Pull request merged**